content/applications/finance/accounting/receivables/customer_payments/recording.rst

```diff
@@ -12,10 +12,47 @@ If a payment is **not linked to an invoice or bill**, the customer has an outsta
 your company, or your company has an outstanding debit with a vendor. You can use those outstanding
 amounts to reduce unpaid invoices/bills.
 
+.. seealso::
+   - :doc:`Internal transfers <../../bank/misc/interbank>`
+   - :doc:`../../bank/reconciliation/use_cases`
+   - `Odoo tutorial on Bank Configuration <https://www.odoo.com/slides/slide/bank-configuration-1880?fullscreen=1>`_
+
+Invoices and bills statuses
+===========================
+
+Different types of status exist in Odoo to help you quickly identify the stage of your transactions.
```

Comment:
```suggestion
Different types of statuses exist in Odoo to help you quickly identify the stage of your transactions.
```

Or keep "status" but then you need to modify it in the title as well

Reply:
Ignoring these changes so far as are part of another doc